mezzanine/core/static/mezzanine/js/admin/content_typed.js

```diff
@@ -0,0 +1,28 @@
+jQuery(function($) {
+    var toggleID = function(id) {
+        // Add or remove the page ID from the cookie IDs string.
+        var index = $.inArray(id, ids.split(','));
+        if (index == -1) {
+            if (ids) {ids += ',';}
+            ids += id;
+        }
+        document.cookie = cookie + '=' + ids + '; path=/';
+    };
+
```

Comment:
This still doesn't seem right. `toggleID` has nothing to do with the new "content typed" stuff, it's purely for pages and page_tree.js

The JS for content typed stuff should literally be a one-line function that probably doesn't even warrant its own file, eg:

`location.href = this[this.selectedIndex].value;`

The tricky part is how page_tree.js then integrates with that so it can run `toggleID` - I'm not sure about the best way to do that, and what order things occur in with jQuery when dealing with multiple event handlers, but there must be some sane way to deal with that.